pkg/cmd/task/task_download.go

```diff
@@ -0,0 +1,174 @@
+package task
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-core-cli/pkg/mesos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/spf13/cobra"
+)
+
+const description = `Downloads files from the sandbox of a given task.
```

Comment:
I don't think we commonly use a `const` for the description, was this done because it is a longer piece of text than usual?

Reply:
Yes, using it a multiline string directly in the `cmd` struct looked pretty ugly.